packages/cyberstorm/src/components/PackageCard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/PackageCard.module.css";
+import { MetaItem } from "./MetaItem";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faDownload,
+  faThumbsUp,
+  faHardDrive,
+} from "@fortawesome/free-solid-svg-icons";
+import { Tag } from "./Tag";
+
+const defaultImageSrc = "";
+
+export interface PackageCardProps {
+  packageName: string;
+  description?: string;
+  imageSrc?: string;
+  downloadCount: string;
+  likes: string;
+  size: string;
+  author?: string;
+  link: string;
+  lastUpdated?: string;
+  isPinned?: boolean;
+  isNsfw?: boolean;
+  isDeprecated?: boolean;
+  packageCardStyle: "default";
+  categories: string[];
+}
+
+/**
+ * Cyberstorm PackageCard component
+ */
+export const PackageCard: React.FC<PackageCardProps> = (props) => {
+  const {
+    packageName,
+    description,
+    imageSrc,
+    downloadCount,
+    likes,
+    size,
+    author,
+    link,
+    lastUpdated,
+    isPinned,
+    isNsfw,
+    isDeprecated,
+    packageCardStyle,
+    categories,
+  } = props;
+
+  const authorLink = ""; //TODO: author link
+
+  return (
+    <div
+      /* TS is not aware of defaultProps of function components. */
+      /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
+      className={`${styles.root}
+      ${getStyle(packageCardStyle)}`}
+    >
+      <div className={styles.imageWrapper}>
+        <img
+          src={imageSrc ? imageSrc : defaultImageSrc}
+          className={styles.image}
+          alt={"package card of" + packageName}
+        />
+      </div>
+
+      <div className={styles.content}>
+        <a href={link} className={styles.name}>
```

Comment:
So this is the same discussion we've been having with Oksamies in [the breadcrumbs PR](https://github.com/thunderstore-io/thunderstore-ui/pull/572). We don't want to use `<a>` elements directly in the components, since e.g. in the website we want them to be Next.js's link components to ensure smooth transitions between pages. To tackle this and some other issues, there's the LinkProvider pattern that should be used instead ([example](https://github.com/thunderstore-io/thunderstore-ui/blob/master/packages/components/src/components/PackageCard.tsx#L92)).

Given that Oksamies is currently working on that, you should probably wait for that to get done to avoid duplicate work. But it also means that if this is merged as is, we need tickets reminding us to fix this.

Reply:
Added a //TODO to the code, I'll also add a ticket about this